time-series/time-series-api/src/main/java/com/powsybl/timeseries/TimeSeriesTable.java

```diff
@@ -534,15 +537,33 @@ private static BufferedWriter createWriter(Path file) throws IOException {
 
     public void writeCsv(Path file) {
         try (BufferedWriter writer = createWriter(file)) {
-            writeCsv(writer, TimeSeriesConstants.DEFAULT_SEPARATOR, ZoneId.systemDefault());
+            writeCsv(writer, TimeSeriesConstants.DEFAULT_SEPARATOR, ZoneId.systemDefault(), new CsvParserConfig());
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+    }
+
+    public void writeCsv(Path file, CsvParserConfig csvParserConfig) {
+        try (BufferedWriter writer = createWriter(file)) {
+            writeCsv(writer, TimeSeriesConstants.DEFAULT_SEPARATOR, ZoneId.systemDefault(), csvParserConfig);
         } catch (IOException e) {
             throw new UncheckedIOException(e);
         }
     }
 
     public String toCsvString(char separator, ZoneId zoneId) {
         try (StringWriter writer = new StringWriter()) {
-            writeCsv(writer, separator, zoneId);
+            writeCsv(writer, separator, zoneId, new CsvParserConfig());
+            writer.flush();
```

Comment:
I wonder if the flush should be moved in the writeCsv() method to avoid user to forget it. What do you think?

Reply:
Done